src/content/en/updates/2017/03/devtools-release-notes.md

```diff
@@ -0,0 +1,166 @@
+description: New features and changes coming to DevTools in Chrome 58.
+
+{# wf_updated_on: 2017-03-03 #}
+{# wf_published_on: 2017-03-03 #}
+{# wf_tags: chrome58,devtools #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: New features and changes coming to DevTools in Chrome 58. #}
+
+{% include "web/tools/chrome-devtools/_shared/styles.html" %}
+
+# What's New In DevTools (Chrome 58) {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+Welcome to the first installment of the DevTools release notes! From here on
+out, the first time you open a new version of Chrome, DevTools opens the
+**What's New** drawer with a link to the release notes for that version.
+
+## Highlights
+
+* The Timeline panel has been renamed to the Performance panel.
+* The Profiles panel has been renamed to the Memory panel.
+* Cookie values are now editable.
+* DevTools now automatically pauses before out-of-memory errors.
+
+## New Features
+
+### Editable cookies {: #cookies }
+
+Double-click on a cell in the **Cookies** tab to edit that value.
+
+<figure>
+  <img src="/web/updates/images/2017/03/editable-cookies.png"/>
+  <figcaption>
+    <b>Figure 1</b>. Editing a cookie
+  </figcaption>
+</figure>
+
+Thanks to [kdzwinel](https://twitter.com/kdzwinel) for the contribution!
+
+### Inspectable and Editable CSS variables in the Styles pane {: #css-variables }
+
+You can now inspect and edit CSS variables in the Styles pane. See [CSS
+Variables Demo][css vars] to try it out yourself.
+
+[css vars]: https://googlechrome.github.io/devtools-samples/author/css-vars
+
+### Out-of-memory breakpoints {: #out-of-memory-breakpoints }
+
+When an app allocates a lot of memory in a short amount of time, DevTools now
+automatically pauses and increases the heap limit. This enables you to inspect
+the heap, execute commands on the Console to free up memory, and continue
+debugging the issue. See [One Small Step For Chrome, One Giant Heap For
+V8][heap] for more information.
+
+<figure>
+  <img src="/web/updates/images/2017/03/out-of-memory-breakpoint.png"/>
+  <figcaption>
+    <b>Figure 2</b>. Paused on an out-of-memory breakpoint
+  </figcaption>
+</figure>
+
+[heap]: https://v8project.blogspot.com/2017/02/one-small-step-for-chrome-one-giant.html
+
+### Breakpoints on canvas creation {: #canvas-creation-breakpoints }
+
+You can now create [event listener breakpoints][event-listener-breakpoint]
+that are triggered whenever a new canvas context is created.
+
+<figure>
+  <img src="/web/updates/images/2017/03/canvas-breakpoint.png"/>
+  <figcaption>
+    <b>Figure 3</b>. Canvas creation breakpoints via the <b>Create canvas
+    context</b> checkbox in the <b>Event Listener Breakpoints</b> pane
+  </figcaption>
+</figure>
+
+[event-listener-breakpoint]: /web/tools/chrome-devtools/javascript/breakpoints#event-listeners
+
+### Start time stats in the Timing tab {: #start-stats }
```

Comment:
No, sections within a panel that clearly resemble tabbed interfaces are called `tabs`. This is also consistent in the docs.